src/engine/client/cl_cgame.cpp

```diff
@@ -1248,6 +1248,25 @@ void CGameVM::QVMSyscall(int syscallNum, Util::Reader& reader, IPC::Channel& cha
 			});
 			break;
 
+		case CG_R_BATCHINPVS:
+			IPC::HandleMsg<Render::BatchInPVSMsg>(channel, std::move(reader), [this] (
+				const std::array<float, 3>& origin,
+				const std::vector<std::array<float, 3>>& posEntities,
+				std::vector<bool>& inPVS)
+			{
+
+				size_t numEntities = posEntities.size();
+
+				inPVS.reserve(MAX_LOCATIONS);
```

Comment:
Does it makes sense to also do a `inPVS.reserve(MAX_LOCATIONS);` here as `inPVS` is an argument?

I tried it, the code builds and run, but I don't know if that changes something.

Reply:
It doesn't make sense to have both `reserve` and `resize`. You can do either reserve+push_back, or resize+`a[b]=c`.

Reply:
I rewrote that part, in fact I could iterate the source array directly by using `push_back` for the destination, which is cleaner.